indigo_tasks/models.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+
+from indigo_api.models import Country, Locality, Work, Document
+from django.contrib.auth.models import User
+
+import datetime
+
+
+class Task(models.Model):
+    title = models.CharField(max_length=256, null=False, blank=False, default='New task: %s' % (datetime.date.today()))
```

Comment:
`default='New task: %s' % (datetime.date.today())`

This will be evaluated when the app first fires up. So your model will be created with a hard-coded default of `New task: 2018-12-03`. It's easier simply not to have a default :)

Reply:
done